config/locales/shared/balance.en.yml

```
@@ -0,0 +1,10 @@
+en:
+  shared:
+    registrations:
+      balance:
+        heading: "Balance"
+        status: "Status"
+        amount: "Amount in £"
+        overpaid: "There have been an overpayment" # TODO: Check with UCD people
```

I can see the comment that it needs to be checked with UCD, but we can still tweak it before hand

```suggestion
        overpaid: "There has been an overpayment" # TODO: Check with UCD people
```